qiskit/qasm3/__init__.py

```diff
@@ -42,6 +42,23 @@
 
 .. autoexception:: QASM3ExporterError
 
+Experimental features
+---------------------
+
+The OpenQASM 3 language is still evolving as hardware capabilities improve, so there is no final
+syntax that Qiskit can reliably target.  In order to represent the evolving language, we will
+sometimes release features before formal standardisation, which may need to change as the review
+process in the OpenQASM 3 design committees progresses.  By default, the exporters will only support
+standardised features of the language.  To enable these early-release features, use the
+``experimental`` keyword argument of :func:`dump` and :func:`dumps`.  The available feature flags
+are:
+
+.. autoclass:: ExperimentalFeatures
+    :members:
+
```

Comment:
It might be good to have an example here because it might not be obvious to people that the usage for this is something like: `dump(qc, experimental=ExperimentalFeatures.SWITCH_CASE_V1)`

Reply:
Done in 13b75a894.